indico/modules/events/editing/operations.py

```diff
@@ -231,6 +232,14 @@ def delete_tag(tag):
     db.session.delete(tag)
 
 
+def create_new_file_type(event, **data):
+    file_type = EditingFileType(event=event)
+    file_type.populate_from_dict(data)
```

Comment:
I'd specify the list of `keys` like in `update_file_type` to make it clear what args this function accepts when looking at its code.